### PR TITLE
Mod file selection feature and config stabilizations

### PR DESCRIPTION
**Pull request by trey-jones**
- You don't have to restart after initial run anymore - config created by settings module now. Hopefully this makes adding new configs easier.
- Allowed skipping of Windows deps to run on other platforms with warning to console
- Added the ability to switch between mod configs in settings

The first item (config changes) was pretty broad, but I think it's for the better.  Open for discussion as far as I'm concerned.

### CODE REVIEW

**Comment by trey-jones** [comment]
I updated all paths to be absolute, except for the ones in the css file.  The point of this was to allow the script to be run without regard to working directory.  I ended up just changing the working directory before executing the app to solve the css issue, rendering the other changes unnecessary.  I didn't remove them though.  I don't think it hurts anything?  Not sure how this affects the windows installation.

File: src/PainterWidget.py [hunk 1]
```diff
@@ -28,6 +34,10 @@ def __init__(self, painter_widget):
         self.last_window = None
 
     def run(self) -> None:
+        if "GetWindowText" not in dir():
```
Comment (Essyer) [inline src/PainterWidget.py:38]: Doesn't work on Windows. The only element of dir() in my testing was 'self'. If there is no way of getting name of  window with focus on Linux you can check platform instead:
`import platform`
`if platform.system().lower() == 'linux':`
&nbsp;&nbsp;&nbsp;&nbsp;`    self.frames_draw_allowed = True`
&nbsp;&nbsp;&nbsp;&nbsp;`    return`

File: src/SettingsWidget.py [hunk 2]
```diff
@@ -132,7 +140,21 @@ def _update_leagues_combo(self) -> None:
             index += 1
         self.combo_league.setCurrentText(self.league)
 
-    def load_main_widget_y(self) -> int:
+    def _update_mod_file_combo(self) -> None:
```
Comment (Essyer) [inline src/SettingsWidget.py:143]: I'm not sure if you can reproduce it on Linux, but on Windows there is small issue with updating this combo. If you don't move settings widget at all, everything works fine. But if you moved it before changing mod file in combo, on each combo update whole settings widget changes its location a bit.

Comment (Essyer) [reply src/SettingsWidget.py:143]: Bug is coming from parent class DragWidget, nothing wrong here.

File: src/PainterWidget.py (hunk 1 above)
Comment (Essyer) [reply src/PainterWidget.py:38]: I will modify it on master, let me know if my change doesn't work on Linux.

**Pull request merged**